README.md

```diff
@@ -1 +1,90 @@
-# celery-logger
+# Celery Logger
+
+celery-logger is a python library for logging [celery](https://docs.celeryproject.org/en/stable/) events such as tasks received, tasks failed/succeeded and tasks retried, along with the passed args.
```

Comment:
I think:

"along with task args"

Is more explicit and less Portuguese -> English free translation

Reply:
Thanks!